RELEASE.md

```diff
@@ -16,11 +16,20 @@
    "3.0.0.30203"). The last section is in "ymmdd" format, where "y" is
    the number of years since 2018 and "mmdd" the month and day portions
    of the release date (e.g., a release on January 12th, 2022 would be
-   x.y.z.40112). Edit the internal version number in
-   `Source/version.cs`, replace the "Upcoming" header in `RELEASE_NOTES.md`
-   with the the public version number, and add a new "Upcoming" header above it.
-   Push and cut a PR, get it approved,
-   and squash and merge those changes to the `master` branch.
+   x.y.z.40112). Edit the internal version number in the following
+   places:
+
+   * `Source/version.cs`
```

Comment:
I'd add a comment to this file noting that the other two instances have to be kept in sync, just in case.

Reply:
Good thought.